src/main/ts/utils/EditorUtils.ts

```diff
@@ -0,0 +1,17 @@
+import { CallExpression, MemberExpression } from 'estree';
+import { extractIdentifier, extractMemberIdentifiers } from './ExtractUtils';
+
+export const isEditorMemberExpression = (node: MemberExpression) => {
+  const identifier = extractIdentifier(node.object);
+  return identifier && (identifier.name === 'ed' || identifier.name === 'editor');
```

Comment:
So eslint doesn't have type information available that we could inspect here instead of specific names?

Reply:
You can get it to my knowledge... but that requires the types to be parsed and anything with type information is **a lot** slower from what I've seen elsewhere (none of the current use have it). I'm just not convinced it's worth it for this case tbh.